pkg/operator/ceph/config/monstore.go

```diff
@@ -235,19 +263,31 @@ func (m *MonStore) SetAll(clientName string, settings map[string]string) error {
 	cephCmd := client.NewCephCommand(m.context, m.clusterInfo, args)
 
 	out, err := cephCmd.RunWithTimeout(exec.CephCommandsTimeout)
+	fileContent, readErr := os.ReadFile(assimilateConfPath.Name() + ".out")
+	if readErr != nil {
+		logger.Errorf("failed to open assimilate output file %s.out. %v", assimilateConfPath.Name(), readErr)
```

Comment:
Is this file always created even if it's empty or could there be a file not found error?

Reply:
Yes, this files always gets created. 

```
sh-4.4$ ceph config assimilate-conf -i /tmp/input.txt -o /tmp/output.txt
sh-4.4$ cat /tmp/output.txt
sh-4.4$ cat /tmp/input.txt
[global]
rbd_default_map_options = ms_mode=secure
sh-4.4$ 
```